contexts/arbo-context/filter-update-steps/adjust-map-position.tsx

```diff
@@ -35,10 +37,20 @@ const getAllBoundingBoxesFromSelectedFilters = (
       (boundingBox: BoundingBox | undefined): boundingBox is BoundingBox =>
         !!boundingBox
     );
+  const selectedWHORegions = selectedFilters["whoRegion"] ?? [];
+  const boundingBoxesFromSelectedWHORegions = selectedWHORegions
+    .map((unRegion) =>
```

Comment:
❓ This is throwing me for a loop ahaha, unRegion !== whoRegion right? so should this here be .map((whoRegion)) or am I missing something

Reply:
Yup you're 100% correct. Whoops, sorry skimmed this a bit too hard hahaha

f4ae019ee20c9627d84153ec594739b92f3300ab